constants/chatbotCommands.py

```diff
@@ -1761,11 +1761,9 @@ async def mpMake() -> Optional[str]:
             host_user_id=-1,
             is_tourney=True,
         )
-        multiplayer_match = await matchList.getMatchFromChannel(chan)
```

Comment:
@lenforiee - error you noticed was happening in this function -- `chan` will never be the right `#mp_{id}` channel, as it does not exist when the user runs the cmd